pkg/storaged/content-views.jsx

```diff
@@ -0,0 +1,744 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+"use strict";
+
+var cockpit = require("cockpit");
+var dialog = require("./dialog");
+var utils = require("./utils.js");
+
+var React = require("react");
+var CockpitListing = require("cockpit-components-listing.jsx");
+var StorageControls = require("./storage-controls.jsx");
+var FormatDialog = require("./format-dialog.jsx");
+
+var StorageButton = StorageControls.StorageButton;
+var StorageLink =   StorageControls.StorageLink;
+var FormatButton =  FormatDialog.FormatButton;
+
+var FilesystemTab   = require("./fsys-tab.jsx").FilesystemTab;
+var CryptoTab       = require("./crypto-tab.jsx").CryptoTab;
+var BlockVolTab     = require("./lvol-tabs.jsx").BlockVolTab;
+var PoolVolTab      = require("./lvol-tabs.jsx").PoolVolTab;
+var PVolTab         = require("./pvol-tabs.jsx").PVolTab;
+var MDRaidMemberTab = require("./pvol-tabs.jsx").MDRaidMemberTab;
+var PartitionTab    = require("./part-tab.jsx").PartitionTab;
+var SwapTab         = require("./swap-tab.jsx").SwapTab;
+var UnrecognizedTab = require("./unrecognized-tab.jsx").UnrecognizedTab;
+
+var _ = cockpit.gettext;
+var C_ = cockpit.gettext;
+
+function create_tabs (client, target, is_partition) {
```

Comment:
extra whitespace after function name - this is the case in a lot of places in this pull request